HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.Steps.cs

```diff
@@ -186,7 +186,7 @@ public void Then_only_endpoints_with_chosen_parameters_should_be_shown()
             Assert.True(selectedTags.Any(m => string.Equals(m.Text, tag)));
         }
 
-        public void Then_endpoints_with_selected_filters_should_be_shown(int expectedCountOfSelectedTags, int expectedCountOfSelectedStatuses)
+        public void Then_endpoints_should_be_displayed_with_expected_count_of_filters(int expectedCountOfSelectedTags, int expectedCountOfSelectedStatuses)
```

Comment:
Not much have changed here: `6. THEN endpoints should be displayed with expected count of filters [expectedCountOfSelectedTags: "1"] [expectedCountOfSelectedStatuses: "1"] `

Would it not be better to ensure that the step displays something like: `6. THEN endpoints should be filtered with 1 tag and 1 status filters`

`public void Then_endpoints_should_be_filtered_with_TAGFILTERS_tag_and_STATUSFILTERS_status_filters(int tagFilters, int statusFilters) {}`

[see lightbdd wiki](https://github.com/Suremaker/LightBDD/wiki/04-Formatting-Parameterized-Steps)